batch.go

```diff
@@ -16,7 +16,7 @@ import (
 // If readonly is false, you can use Put and Delete method to write data to the batch.
 // The data will be written to the database when you call Commit method.
 //
-// Batch is not a transaction, it does not guarantee isolation.
+// Batch is not a transaction, it does not guarantee isolation(only guarantee read committed).
```

Comment:
It`s not a very accurate description, actually, the isolation is Serializable because we have a global lock.

Reply:
I was thinking about this problem just now, I will delete it.